torchdata/datapipes/iter/transform/callable.py

```diff
@@ -65,6 +96,7 @@ def __init__(
         assert batch_size > 0, "Batch size is required to be larger than 0!"
         self.batch_size = batch_size
         self.input_col = input_col
+        ensure_fn_works(fn, input_col)
```

Comment:
Let's remove this part for now. I need to double check with the behavior of `map_batches`

Reply:
I will take out both the function definition and the checks then until core merges my checks